### PR TITLE
chore(ci): add v8 and v11 of Node version

### PR DESCRIPTION
**Pull request by hiroppy**
and add node_modules as a cache

- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### Motivation / Use-Case

https://github.com/webpack/webpack-dev-server/pull/1650#issuecomment-461023293

### Breaking Changes

no

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,9 +26,12 @@ jobs:
       env: SCRIPT=test
       # Node 10 instead of 'stable' since Node 11 segfaults
```
Comment (hiroppy) [inline .travis.yml]: I would like to erase it if it is stable.

Comment (evilebottnawi) [reply .travis.yml]: @hiroppy let's remove comment and use `node_js: 8`, stable can be changed in future

Comment (evilebottnawi) [reply .travis.yml]: Also let's add node_js: 11

Comment (hiroppy) [reply .travis.yml]: Is it like this?(or we should use `'lts/*'`?)

```
    - &osx
      stage: Test (MacOS)
      os: 'osx'
      env: SCRIPT=test
    - <<: *osx
      node_js: 11
    - <<: *osx
      node_js: 10
    - <<: *osx
      node_js: 8
    - <<: *osx
      node_js: 6
```

Comment (evilebottnawi) [reply .travis.yml]: :+1: Let's avoid `lts/*`, i think travis interpreted first line as test

Comment (hiroppy) [reply .travis.yml]: Updated

**Comment by evilebottnawi** [comment]
God job thanks!

**Comment by evilebottnawi** [comment]
What about solve this https://github.com/webpack/webpack-dev-server/issues/1560? Should be easy :+1: PR already exists need add tests https://github.com/webpack/webpack-dev-server/pull/1562

**Comment by hiroppy** [comment]
I'll try it:)

**Pull request merged**